cidc-api/migrations/versions/37888cac6b2b_.py

```diff
@@ -0,0 +1,40 @@
+"""empty message
```

Comment:
This file was auto-generated by running `FLASK_APP=app.py flask db migrate` in the `cidc-api` directory.